bin/plotting/pycbc_page_foundmissed

```diff
@@ -75,8 +76,19 @@ elif args.far_type == 'exclusive':
     ifar_found = f['found_after_vetoes/ifar_exc'][:]
     far_title = 'Exclusive'
 
-if args.limit_ifar:
-    ifar_found = numpy.minimum(ifar_found, args.limit_ifar)
+upper_lim = False
+lower_lim = False
+if args.ifar_limits:
+    if args.ifar_limits[0] < 0 or args.ifar_limits[1] < 0:
+        parser.error("IFAR limits must be zero (no limit) or positive")
+    if args.ifar_limits[0] > 0:
```

Comment:
I think you don't need this `if` switch .. the lower limit being 0 _is_ no limit.

Reply:
Ah, maybe just to control how the colorbar is plotted